pkg/limitador/k8s_objects.go

```diff
@@ -94,6 +94,7 @@ func LimitadorDeployment(limitador *limitadorv1alpha1.Limitador) *appsv1.Deploym
 							Command: []string{
 								"limitador-server",
 								fmt.Sprintf("%s%s", LimitadorCMMountPath, LimitadorConfigFileName),
+								storageOptions(limitador.Spec.Storage, storageConfigSecret),
```

Comment:
heads up, one string item per command line param. Each string item will be "quoted". 

```go
Command: []string{                          
    "limitador-server",                     
    "/path/file",
    "redis redis://127.0.0.1/a"
},                                          
```
will be run like:

```
docker run IMAGE limitador-server /path/file 'redis redis://127.0.0.1/a'
```
And that would not work. you want
```go
Command: []string{                          
    "limitador-server",                     
    "/path/file",
    "redis",
    "redis://127.0.0.1/a"
},                                          
```